<launch>
    <include file="$(find pamaral_models)/launch/right_hand_keypoints_detection.launch">
        <arg name="input_image_topic" value="front_camera/color/image_raw"/>
    </include>
    
    <node name="dataset_processing" pkg="pamaral_models" type="dataset_processing.py" output="screen">
        <param name="input_folder" value="$(find data)/bag_files"/>
        <param name="output_folder" value="$(find data)/pose_keypoints"/>
    </node>
</launch>
